src/utils/nonNull.ts

```diff
@@ -36,8 +36,8 @@ export function nonNullOrEmptyValue(
 ): string {
   if (!value) {
     throw new Error(
-      l10n.t('Internal error: Expected value to be neither null, undefined, nor empty') +
-        (propertyNameOrMessage ? `: ${propertyNameOrMessage}` : ''),
+      `${errorMessage.internalErrorNullOrUnderinedOrEmpty} +
+      ${propertyNameOrMessage ? `: ${propertyNameOrMessage}` : ''}`,
```

Comment:
We could use [Nullish Coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)